src/vcpkg/base/files.cpp

```diff
@@ -2936,7 +2936,8 @@ namespace vcpkg
             DWORD length_without_null = GetTempPathW(MAX_PATH + 1, temp_folder);
             Path temp_folder_path = Path(Strings::to_utf8(temp_folder, length_without_null)) / "vcpkg";
 #else  // ^^^ _WIN32 // !_WIN32 vvv
-            const Path temp_folder_path = "/tmp/vcpkg";
+            const Path temp_folder_path =
```

Comment:
Does this TMPDIR variable come from POSIX or XDG or similar or is this something made up here?

Reply:
https://en.wikipedia.org/wiki/TMPDIR

Reply:
😅 Thanks!